File: Diploma.Server/Models/LLMExpert.cs
using System.ComponentModel.DataAnnotations;

namespace Diploma.Server.Models
{
    public class LLMExpert
    {
        [Key]
        public int ExpertId {  get; set; }
        public required string Name { get; set; }
        public required string URL { get; set; }
        public ICollection<ExpertEvaluation>? ExpertEvaluations { get; set; }
    }
}

File: diploma.client/src/models/LLMExpert.ts
import ExpertEvaluation from "./ExpertEvaluation";

export default class LLMExpert {
    expertId: number;
    name: string;
    url: string;
    expertEvaluation: ExpertEvaluation[];

    constructor() {
        this.expertId = 0;
        this.name = "";
        this.url = "";
        this.expertEvaluation = [];
    }
}